specification/resources/monitoring/monitoring_create_destination.yml

```
@@ -0,0 +1,41 @@
+operationId: monitoring_create_destination
+
+summary: Create Logging Destination
+
+description: To create a new destination, send a POST request to `/v2/monitoring/sinks/destinations`.
+
+tags:
+  - Monitoring
+
+requestBody:
+  required: true
+  content:
+    application/json:
+      schema:
+        $ref: 'models/destination_request.yml'
```

```suggestion
      schema:
        $ref: 'models/destination_request.yml'

      examples:
        Managed Opensearch Cluster:
          value:
            name: managed_opensearch_cluster
            type: opensearch_dbaas
            config:
              endpoint: db-opensearch-nyc3-123456-do-user-123456-0.g.db.ondigitalocean.com
              cluster_uuid: 85148069-7e35-4999-80bd-6fa1637ca385
              cluster_name: managed_dbaas_cluster
              index_name: logs
              retention_days: 14

        External Opensearch Cluster:
          value:
            name: external_opensearch
            type: opensearch_ext
            config:
              endpoint: example.com
              credentials:
                username: username
                password: password
              cluster_name: external_opensearch
              index_name: logs
              retention_days: 14

```

When there are two distinct types with slightly different requests, it can be nice to provide specific examples for each. Guessing a bit at these.